src/coreclr/vm/callconvbuilder.cpp

```diff
@@ -60,7 +60,7 @@ namespace
     {
 #define BASE_CALL_CONV(name, flag) { \
         MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name), \
-        STRING_LENGTH(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name)), \
+        STRING_LENGTH(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name)) - 1, \
```

Comment:
nit: we can use `ARRAY_SIZE(MAKE_FULLY_QUALIFIED_CALLCONV_TYPE_NAME_PREFIX(name))` (without -1) for these, since `STRING_LENGTH` is defined as: https://github.com/dotnet/runtime/blob/69679789fba3c377b1d14689ebab2e30a9f9ad8a/src/native/minipal/utils.h#L10